tools/llvm-vr-dump.cpp

```diff
@@ -41,24 +41,21 @@
 #endif
 
 #undef NDEBUG // we need dump methods
-#include "dg/llvm/ValueRelations/ValueRelations.h"
+#include "dg/llvm/ValueRelations/GraphBuilder.hpp"
+#include "dg/llvm/ValueRelations/StructureAnalyzer.hpp"
+#include "dg/llvm/ValueRelations/RelationsAnalyzer.hpp"
```

Comment:
Now I realized that you use `.hpp` suffix. I have nothing against it in general, but in DG all the headers use the suffix `.h`. Change it please, having some headers as `.h` and some as `.hpp` would be confusing.